.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,113 @@
+          required: true
+
+  - type: input
+    id: app-version
+    attributes:
+     label: Affected version
+     description: "In which version did you encounter the bug?"
+     placeholder: "x.xx.x - Can be seen in the app under the 'About'-section"
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to reproduce the bug
+      description: |
+        What did you do for the bug to show up?
+        
+        If you can't cause the bug to show up again reliably (and hence don't have a proper set of steps to give us), please still try to give as many details as possible on how you think you encountered the bug.
+      placeholder: |
+        1. Go to '...'
+        2. Press on '....'
+        3. Swipe down to '....'
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: |
+        Tell us what you expect to happen.
+
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: Actual behavior
+      description: |
+        Tell us what happens with the steps given above.
+
+  - type: textarea
+    id: screen-media
+    attributes:
+      label: Screenshots/Screen recordings
+      description: |
+        A picture or video is worth a thousand words.
+        
+        If applicable, add screenshots or a screen recording to help explain your problem.
+        GitHub supports uploading them directly in the text box.
+        If your file is too big for Github to accept, try to compress it (ZIP-file) or feel free to paste a link from an image/video hoster here instead.
+        
+        :heavy_exclamation_mark: DON'T POST SCREENSHOTS OF THE ERROR PAGE.
+        Instead paste the text in the "Logs"-section below.
+
+  - type: textarea
+    id: logs
+    attributes:
+      label: Logs
+      description: |
+        If your bug includes a crash (where you're shown the Error Report page with a bunch of info), tap on "Copy formatted report" at the bottom and paste it here.
+
+  - type: input
+    id: device-os-info
+    attributes:
+     label: Affected Android/Custom ROM version
+     description: |
+      With which operating system (+ version) did you encounter the bug?
+     placeholder: "Example: Android 12 / LineageOS 18.1"
+
+  - type: input
+    id: device-model-info
+    attributes:
+     label: Affected device model
+     description: |
+      With which device did you encounter the bug?
+     placeholder: "Example: Huawei P20 lite (ANE-LX1) / Samsung Galaxy S20"
+
+  - type: textarea
+    id: additional-information
+    attributes:
+      label: Additional information
+      description: |
+        Any other information you'd like to include, for instance that 
+        * the affected device is a TV or foldable
```

Not sure about this, but here goes:
```suggestion
        * the affected device is a TV or foldable device
```